src/main/kotlin/com/learnspigot/bot/help/CloseCommand.kt

```diff
@@ -38,16 +38,22 @@ class CloseCommand {
             return
         }
 
+        event.deferReply().queue()
+
         val contributors: List<Member> =
-            channel.retrieveThreadMembers().complete().asSequence().filter { member: ThreadMember -> member.id != channel.ownerId }
+            channel.retrieveThreadMembers().complete().asSequence()
+                // excludes the author of the channel
+                .filter { member: ThreadMember -> member.id != channel.ownerId }
+                // excludes bots
                 .filter { member: ThreadMember -> !member.user.isBot }
-                .filter{ member: ThreadMember ->
+                // excludes users that hasn't sent a single message to this channel (i.e: users that clicked the 'follow post' button)
+                .filter { member: ThreadMember ->
                     val messageHistory = channel.iterableHistory.complete()
                     messageHistory.any { it.author.id == member.id }}
                 .take(25).map { it.member }.toList()
 
         if (contributors.isEmpty()) {
-            event.replyEmbeds(
+            event.hook.sendMessageEmbeds(
```

Comment:
Since we relocated `event.deferReply().queue()` to an earlier position, we have to respond via the hook, otherwise we get an exception (`This interaction has already been acknowledged or replied to. You can only reply or acknowledge an interaction once!`)